types.go

```diff
@@ -91,6 +91,7 @@ type Faceting struct {
 
 type Embedder struct {
 	Source           string `json:"source"`
+	URL              string `json:"url"`
```

Comment:
Please add an `omitempty` tag, url is optional for ollama.

Reply:
:+1: